tests/Datagrid/DatagridMapperTest.php

```diff
@@ -225,6 +225,9 @@ public function testAddRemove(): void
         $this->assertSame('fooFilterName', $fieldDescription->getOption('field_name'));
     }
 
+    /**
+     * @psalm-suppress InvalidScalarArgument
```

Comment:
what does this mean?

Reply:
```
$this->datagridMapper->add(12345);
```
Is invalid. The phpdoc is asking for `FieldDescriptionInterface|string`  but we're voluntary passing a wrong argument.